src/app/add-modules/add-module-modal/add-module-modal.component.scss

```diff
@@ -0,0 +1,25 @@
+@import "/src/assets/styles/variables";
+@import "/src/assets/styles/mixin";
+
+::ng-deep.mat-dialog-container {
+  border-top: 4px solid $primary;
+}
+
+
+.add-dialog {
+  font-family: $font-family;
+  border-radius: 4px 4px 0px 0px;
+
+  .add-dialog__content-input {
+    border: none;
+    outline: none;
+  }
+
```

Comment:
create one file overwrite.scss and overwrite border-radius of mat-button to 4px in there. And remove the following codes(line 18 to 24)